build-tools/debian-metadata/xamarin.android.install

```diff
@@ -0,0 +1,2 @@
+bin/Debug/bin/xabuild		/usr/bin
```

Comment:
It's not quite a matter of "debug symbols"; at least for managed code: we *always* produce `.pdb` files, even in the Release configuration, and thus we can redistribute them in the package. (At present, they *will* be redistributed.)

The primary difference is instead on whether *optimizations* are applied: using `csc /optimize+` for managed code, and whether we use `-O0 -g` (debug) or `-O2` (release) for native code.

I thus think we should distribute the `bin/Release` binaries.

Unfortunately this somewhat complicates the "local Linux build" scenario; you'd need to build with:

    make prepare all package-deb CONFIGURATION=Release

which might not be desirable. Is there any way to instead *generate* the `build-tools/debian-metadata/xamarin.android.install` file, and use the value of `$(CONFIGURATION)`/`$(lastword $(CONFIGURATIONS))`?